### DISCUSSION TITLE
Welcome to Jolokia Discussions!

### DISCUSSION THREAD

**Discussion by tadayosi** [rhuss/jolokia#551] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by grgrzybek** [comment]
I'm Grzegorz Grzybek working mainly with OSGi projects.

In Jolokia I helped with JDK17 issues - mostly related to removing direct access to classes/packages which are not _open_. One of them was self-signed certificate generation which was using private `sun.security.tools.keytool.CertAndKeyGen`

When working on javax→jakarta transition in various OSGi projects (like https://github.com/ops4j/org.ops4j.pax.web), I'm soon going to review OSGi bits in Jolokia (which currently use ancient `org.osgi.service.http.HttpService` - I will switch to Whiteboard approach.

**Comment by tadayosi** [comment]
Hi community,

I am Tadayoshi Sato, working on [Hawtio](https://github.com/hawtio/hawtio) project.

I'm currently working on the next generation version of Hawtio, and since Jolokia is the essential building block for Hawtio, I'd like to keep Jolokia alive and active. Especially, I'm really interested in migration to Jakarta EE 10 for Jolokia, so that it can be up-to-date and kept using in the latest versions of popular frameworks such as Spring Boot 3 and Quarkus 3.
